mm2src/coins/utxo/utxo_common.rs

```diff
@@ -1603,16 +1597,16 @@ pub async fn send_maker_spends_taker_payment<T: UtxoCommonOps + SwapOps>(
         coin.get_htlc_spend_fee(DEFAULT_SWAP_TX_SPEND_SIZE, &FeeApproxStage::WithoutApprox)
             .await
     );
-    if fee >= payment_value {
+    if fee.fee >= payment_value {
```

Comment:
when I was studying mm2 code I messed up with used several 'fee' concepts. So I'd suggest we use 'qualified' names for fees, if possible, like 'tx_fee' or 'dex_fee' (not just 'fee') to easy get what fee is meant